.github/workflows/security-scans.yml

```diff
@@ -0,0 +1,226 @@
+name: "Security Scans"
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 3 * * 1' # run tests at 1 AM (UTC), every monday (1)
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  GO_VERSION: "~1.19"
+
+jobs:
+  prepare-security-scans:
+    name: "Prepare Security Scans"
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Find latest successful run ID
+        id: last_run_id
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          RUN_ID=$(\
+            curl -sL \
+              -H 'Accept: application/vnd.github.v3+json' \
+              -H "Authorization: token $GITHUB_TOKEN" \
+              "api.github.com/repos/${{ github.repository }}/actions/workflows/CI.yaml/runs?branch=main" | \
+            jq '[.workflow_runs[] | select(
+              (.head_commit != null) and ( .conclusion == "success" ) 
+            )][0] | .id')
+          echo "Run ID that will be used to download artifacts from: $RUN_ID"
+          echo "RUN_ID=$RUN_ID" >> $GITHUB_OUTPUT
```

Comment:
since this part is used multiple times already, we could put it into a separate action and re-use it?

Reply:
ah yeah i thought we have another occurrence in KLT already. then it should be fine to leave it.
or you can pull it out and put the action into keptn/gh-automation and then re-use it here and in keptn/keptn

Reply:
Extraction into `keptn/gh-automation` can be handled in a follow up PR